index.html

```diff
@@ -74,9 +74,9 @@ <h1 class="sr-only">If These Lands Could Talk</h1>
 
 							<!-- Page Links -->
 							<ul id="nav-links" class="nav-links">
-								<li><a href="#">About US</a></li>
+								<li><a href="./pages/about.html">About US</a></li>
```

Comment:
@SolivanLau sorry I missed this point before, but all of our pages should be in the root of the document, next to index.html. That way, we don't have to have `itlct.org/pages/about.html` in the url, we can just have `itlct.org/about.html`. 

This navigation breaks on these pages because it's adding an extra `/pages/` to a url that's already in the pages folder. This is what I get when I click on it: `http://127.0.0.1:5500/pages/pages/about.html`.

Please move this page to the root, update your css/js paths, and change the navigation href to `/about.html`.

Reply:
Sounds good. Since each page is currently like this, I am thinking adjusting page's location according to its PR will be  the easiest